src/main/resources/META-INF/plugin.xml

```diff
@@ -211,6 +211,21 @@
             <!-- In $default keymap `control` key (Windows/Linux) corresponds to `cmd` key on macOS -->
             <keyboard-shortcut keymap="$default" first-keystroke="control alt shift M" />
         </action>
+
+        <action id="GitMachete.CheckoutNextAction"
+                class="com.virtuslab.gitmachete.frontend.actions.navigation.CheckoutNextAction">
+            <keyboard-shortcut keymap="$default" first-keystroke="alt DOWN" />
+        </action>
+
```

Comment:
Also one more thing: when Git tool window is opened but Git Machete tab is not, do the shortcuts take effect or not?

Reply:
No they don't.

Reply:
Okay, gr8